Tests/AlicerceTests/DeepLinking/TreeRouterTests.swift

```diff
@@ -1115,7 +1116,7 @@ class TreeRouterTests: XCTestCase {
         let expectation = self.expectation(description: "TreeRouter.route")
         defer { waitForExpectations(timeout: expectationTimeout, handler: expectationHandler) }
 
-        testHandler.didCallHandle = { (url, parameters, queryItems) in
+        let completion: (HandledRoute) -> Void = { (url, parameters, queryItems) in
```

Comment:
I don't think we need the brackets around the parameter list for the block? 😕 

Same comment for all the blocks below too 😄